src/_guides/libraries/objective-c-interop.md

```diff
@@ -11,6 +11,11 @@ running on the [Dart Native platform](/overview#platform), on macOS or iOS,
 can use `dart:ffi` and [`package:ffigen`]({{page.ffigen}})
 to call Objective-C APIs.
 
+{{site.alert.note}}
+  **Note**: This interop feature is **experimental**,
+  and [in active development][https://github.com/dart-lang/sdk/issues/49673].
```

Comment:
Bad link syntax. Use `()` not `[]` around the URL.

Reply:
```suggestion
  and [in active development](https://github.com/dart-lang/sdk/issues/49673).
```